BerghAdmin/Services/UserManagement/IUserService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using BerghAdmin.Authorization;
+using Microsoft.AspNetCore.Identity;
+using System.Security.Claims;
+
+namespace BerghAdmin.Services.UserManagement;
+
+public interface IUserService
+{
+    Task DeleteUserAsync(string naam);
```

Comment:
Ik vind dat we dat achtervoegsel Async niet meer hoeven te doen. Alles is Async hedentendage (schrijf je dat zo?).